backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -1811,6 +1815,28 @@ private static Predicate getFuzzyUsersQueryPredicate(
     return null;
   }
 
+  public static Set<String> getResourceIdsBasedOnUserInfo(
```

Comment:
```suggestion
  public static Set<String> getResourceIdsFromUserWorkspaces(
```
Let's give things very explicit names

Reply:
Renamed as per the suggestion